### PR TITLE
Add a handrawn tile and add music into it, Add a background audio

### PR DESCRIPTION
**Pull request by STEdirisuriya**
### When you change [things like this in brackets] please remember to delete brackets after you change it.

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood [see CONTRIBUTING.md](https://github.com/fossasia/labyrinth/blob/master/CONTRIBUTING.md)
- [x] Included a Preview link and screenshot showning after and before the changes.
- [x] Included a description of change below.
- [x] Squashed the commits.

# Changes done in this Pull Request

- [Preview Link](http://rawgit.com/STEdirisuriya/labyrinth/master/index.html)
, 

# The problem(s) I want to solve or The facilitie(s) I want to improve is/are,
I add a tile. and add a music into that. Add a background audio

### Screenshots,
tile

### CODE REVIEW

File: js/tiles.js [hunk 1]
```diff
@@ -1252,5 +1252,29 @@ const Superhero = {
...
+        canLeaveToTheLeft() {
+            return false;
+        },
+        createImages: function () {
+            this.wallTop = this.createImage("tiles/rooms/door/top.svg");
+            this.wallRight = this.createImage("tiles/rooms/door/right.svg");
+            this.ground = this.createImage("tiles/rooms/floor/art.svg");
+        },
+        visit: function () {
+            playAudio("audio.mp3");
```
Comment (0x48piraj) [inline js/tiles.js]: It's the wrong path. Please correct the path!

**Comment by samyakbvs** [comment]
Pls provide a screenshot of your tile

File: js/levels.js [hunk 2]
```diff
@@ -14,7 +14,7 @@ function createFirstLevel()
     [door.none, door.minecraftEntry, door.minecraft, door.none, door.goal, door.none],
     [door.none, door.both, door.Forbidden, door.none, door.none, door.none],
     [door.none, door.sofa, door.outdoor, door.chessMate, door.texture, door.chessStale],
-    [door.none, door.yellowBoxes, PlayerStartsAt(door.black), door.green, door.banner, door.threeHeads],
+    [door.none, door.art, PlayerStartsAt(door.black), door.green, door.banner, door.threeHeads],
```
Comment (abishekvashok) [inline js/levels.js]: `yellowBoxes` are'nt implemented again so can you pls remove any other tile which is implemented twice?

Comment (STEdirisuriya) [reply js/levels.js]: Yellow boxes tile not shown, that's why I removed it

Comment (STEdirisuriya) [reply js/levels.js]: image

File: js/sound.js [hunk 3]
```diff
@@ -55,7 +55,14 @@ const backgroundAudio = [
         legalNotice: "Music by Saarthak Chaturvedi",
         link: "http://github.com/saarthakchats",
     },
-        {
+    {
+        filename: "audio/art/musicofart-backgvideo.mp3",
```
Comment (0x48piraj) [inline js/sound.js]: Now it's good but audio.mp3 is of no use, please delete it. :

Comment (STEdirisuriya) [reply js/sound.js]: No audio.mp3 file

Comment (STEdirisuriya) [reply js/sound.js]: I removed it already

Comment (arjunpandey2003) [reply js/sound.js]: Its still there

Comment (0x48piraj) [reply js/sound.js]: Yep

**Comment by STEdirisuriya** [comment]
I did three changes

**Comment by STEdirisuriya** [comment]
I added tile sound as well

File: js/sound.js (hunk 3 above)
Comment (STEdirisuriya) [reply js/sound.js]: @arjunpandey2003 no there are two sound files beacuse I added background music and tile sound both please see the caption.

File: audio/art/art.license [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+The music was downloaded from Youtube
+
+# License Agreement Explained.
+What You Can Do anything with Our Sounds:
+You can do anything with our sound file.
+We don't have any responsibility about this sound file.
+ 
+Email : supunedirisuriya2002@gmail.com
+Github : STEdirisuriya
```
Comment (niccokunzmann) [inline audio/art/art.license]: I think, the license which reflects your intention most would be public domain.
Just for the record, in Germany, you can not give away all your rights and public domain is not possible. As such one would CC0.

**Comment by niccokunzmann** [comment]
Thanks!

**Pull request merged**